thorlcr/master/thmastermain.cpp

```diff
@@ -1075,41 +1058,44 @@ int main( int argc, const char *argv[]  )
             }
         }
 
-        if (doWorkerRegistration && registry->connect(numWorkers))
+        registry->connect(numWorkers);
+        if (!isContainerized())
         {
+            // bare-metal - check health of dafilesrv's on the Thor cluster.
             if (globals->getPropBool("@replicateOutputs")&&globals->getPropBool("@validateDAFS",true)&&!checkClusterRelicateDAFS(queryNodeGroup()))
             {
                 FLLOG(MCoperatorError, thorJob, "ERROR: Validate failure(s) detected, exiting Thor");
                 return globals->getPropBool("@validateDAFSretCode"); // default is no recycle!
```

Comment:
Would be clearer if this explicitly returned an integer error code.  E.g.

return globals->getPropBool("@validateDAFSretCode") ? TEC_DAFSdown : 0);